pkg/helpers/build.go

```diff
@@ -102,7 +112,8 @@ func NewBakeRequest(project string, opts map[string]buildx.Options, features Usi
 			Load:       features.Load,
 			Save:       features.Save,
 			Lint:       features.Lint,
-			TargetName: &name,
+			SaveTags:   targetSaveTags,
```

Comment:
I believe that this here should pass the `SaveTags` array directly onwards to the API and should not add the target names yet, because later in the build process, after the API returns the `AdditionalTags` field, we then add a suffix to each tag that the API returns (in https://github.com/depot/cli/blob/main/pkg/registry/cli.go). 

We'll want to avoid adding the suffixes here so that we don't end up adding the target suffixes twice:

```suggestion
			SaveTags:   features.SaveTags,
```

Reply:
ah yea, that makes sense!